### ISSUE TITLE
There are no .pdb files in the nuget package

### ISSUE THREAD

**Issue by Holo-Krzysztof** [openpeer/ortc-lib-winrt#2]
...and that makes debugging pretty much impossible when the native library crashes.
It'd help me a lot if these were included.

**Comment by jamescadd** [comment]
Yes it does, sorry about that.  We've looked into this before and will include them in future packages.  Which version are you working against?  Depending on version we may have them archived, in which case I'll get you the files.

**Comment by Holo-Krzysztof** [comment]
It's version 1.0.54.6.

**Comment by jamescadd** [comment]
I'm not seeing archives for that version unfortunately. If you want to build it yourself pull this branch: https://github.com/ortclib/ortclib-sdk/tree/releases/m54
